### ISSUE TITLE
WorkingDirectory with multiple checkouts

### ISSUE THREAD

**Issue by erwin-faceit** [LoremFooBar/ReportGenerator.AzureBlobHistoryStorage#1]
When using this plugin in Azure Devops, it's possible that the working directory differs from the source directory, resulting in an error:

```
2022-12-15T13:20:15: Creating history report
2022-12-15T13:20:15: Error during saving historic report '2022-12-15_13-20-13_CoverageHistory.xml': Underlying process reported a non-zero exit code (128).

Command:
  git rev-parse --show-toplevel

Standard error:
  fatal: not a git repository (or any of the parent directories): .git

You can suppress this validation by calling `WithValidation(CommandResultValidation.None)` on the command.
```

I think it should be possible to specify the source directory using arguments.

**Comment by LoremFooBar** [comment]
Thanks for reporting! Would be great if you can show me how to reproduce. Admittedly I have mostly taken in account specific needs for a project I was working on, so it makes sense that some options may be missing.

**Comment by erwin-faceit** [comment]
When your pipeline starts with something like this

```
resources:
  containers:
  - container: mssql
    image: mcr.microsoft.com/mssql/server:2019-latest
    ports: 
      - 1433:1433
    env:
      ACCEPT_EULA: Y
      SA_PASSWORD: 'r9jwj9*FwXUjqd7#N6g#'
    options: --name mssql
  repositories:
  - repository: self
    type: git
    path: Module
  - repository: Test
    type: git
    name: Test
    path: Test

steps:
  - checkout: self

  - checkout: Test
```

the files will be checked out in specific folders instead of in $(Build.SourcesDirectory). The ```path``` value will be used as a directory name, so the files will be in $(Build.SourcesDirectory)/Test and $(Build.SourcesDirectory)/Module

I've created a fork with a change in it, and will create a pull request right after posting this. Since I'm not using repositories with a single checkout, you might want to check if that still works.

**Comment by LoremFooBar** [comment]
Fixed in #2
